src/pages/404.astro

```diff
@@ -0,0 +1,24 @@
+---
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout title="Page not found" description="Sorry, this page couldn't be found.">
+	<main class="flex flex-col items-center justify-center min-h-screen bg-gray-50 dark:bg-gray-900">
```

Comment:
i deffinetely need to run prettier for this lol. a lot of this code is using tabs.